modules/jcommunity/install/install.php

```diff
@@ -250,4 +253,36 @@ protected function insertUsers(InstallHelpers $helpers, EntryPoint $entryPoint,
             }
         }
     }
+
+    /**
+     * Parse the password field value.
+     * 
+     * @param string $password The password field value ('__empty', '__to_encrypt:`password`' or the encrypted password)
+     * @param dbAuthDriver $driver The database driver
+     * 
+     * @return string The password value to put in the db
+     */
+    protected function parsePassword($password, $login, $driver)
+    {
+        if (strncmp('__', $password, 2)) {
```

Comment:
I think the test is bad... Here, if the password begin with `__`, we return the password, although it should be the contrary.

i prefer `strpos('__', $password) !== 0`, it's more understandable IMHO.

Reply:
sorry, your test is good... forget